packages/chakra-ui/src/components/ui/select.tsx

```diff
@@ -0,0 +1,170 @@
+import { forwardRef, RefObject } from 'react';
+import type { CollectionItem } from '@chakra-ui/react';
+import { Select as ChakraSelect, Portal } from '@chakra-ui/react';
+import { CloseButton } from './close-button';
```

Comment:
Please add a blank line between package-level and local imports
```suggestion
import { Select as ChakraSelect, Portal } from '@chakra-ui/react';

import { CloseButton } from './close-button';
```